nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -197,35 +202,87 @@ define([
         }
 
         /**
-         * TODO
-         * This evaluates the state of the app configuration. If it's ready to go, then we can build the Python
-         * code and prep the app for launch. If not, then we shouldn't, and, in fact, should clear the Python code
-         * if there's any there.
-         * @param {boolean} isError - should be truthy if there's currently a known error in the app param formulation
+         * This evaluates the state of the app configuration. If it's ready to go, then we can
+         * build the Python code and prep the app for launch. If not, then we shouldn't, and,
+         * in fact, should clear the Python code if there's any there.
+         * @param {boolean} fileType - the file type to evaluate app params for
          */
-        function evaluateAppConfig(isError) {
+        function evaluateAppConfig(fileType) {
             /* 2 parts.
              * 1 - eval the set of parameters using something in the spec module.
              * 2 - eval the array of file inputs and outputs.
              * If both are up to snuff, we're good.
              */
+            const otherParamIds = model.getItem(['app', 'otherParamIds', fileType]),
+                paramValues = model.getItem(['params', fileType, PARAM_TYPE]),
+                filePathIds = model.getItem(['app', 'fileParamIds', fileType]),
+                filePathValues = model.getItem(['params', fileType, FILE_PATH_TYPE]);
+
+            const filePathValidations = filePathValues.map((filePathRow) => {
+                return spec.validateParams(filePathIds, filePathRow);
+            });
+            return Promise.all([
+                spec.validateParams(otherParamIds, paramValues),
+                ...filePathValidations,
+            ]).then((results) => {
+                let isValid = true;
```

Comment:
This is a list of objects, representing the validation results from each row, and from the set of non-file-path parameters. If any of those params aren't valid, return 'incomplete'